lib/glific/templates.ex

```diff
@@ -182,6 +190,31 @@ defmodule Glific.Templates do
      ]}
   end
 
+
+  defp contains_invalid_chars?(text) do
+    contains_variable?(text) or
+      contains_newline?(text) or
+      contains_formatting?(text)
+      # contains_emoji?(text) or
```

Comment:
Add or after contains_formatting?(text) and remove the or after contains_emoji?(text)